File: Calculadora.js
let first =  Number(prompt("Digite o primeiro número: "));
let second = Number(prompt("Digite o segundo número: "));
let operation = prompt("Digite a operação (+, -, *, /):");

switch (operation) {
    case "+":
        alert(`O resultado da soma é: ${first + second}`);
        break;
    case "-":
        alert(`O resultado da subtração é: ${first - second}`);
        break;
    case "*":
        alert(`O resultado da multiplicação é: ${first * second}`);
        break;
    case "/":
        if (second !== 0) {
            alert(`O resultado da divisão é: ${first / second}`);
        } else {
            alert("Erro: Divisão por zero não é permitida.");
        }
        break;
    default:
        alert("Operação inválida. Por favor, escolha entre +, -, *, /.");
}

File: Calculadora.py
first = float(input("Digite o primeiro número: "))
second = float(input("Digite o segundo número: "))  
operation = input("Digite a operação (+, -, *, /): ")
if operation == "+":
    result = first + second
    print("O resultado é: ", result)
elif operation == "-":
    result = first - second
    print("O resultado é: ", result)
elif operation == "*":
    result = first * second
    print("O resultado é: ", result)
elif operation == "/":
    result = first / second
    print("O resultado é: ", result)
else:    print("Operação inválida")